internal/app/repository/crudrepositoryimpl_test.go

```diff
@@ -0,0 +1,49 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+	"log"
+	pb "taskForPr/proto"
+	"testing"
+)
+
+func setupDb() *sql.DB {
+	db, err:= sql.Open("postgres", "postgresql://postgres:qwe@localhost:5433/cruddb?sslmode=disable")
+	if err != nil{
+		log.Fatal(err)
```

Comment:
```suggestion
		return fmt.Errorf("sql.Open failed: %w", err)
```
No point to use logger here, You can `log.Fatal` on the caller side.